src/content/development/contribution-guide/_index.en.md

```diff
@@ -58,7 +58,35 @@ To be expanded
 
 #### Pre-Planning
 
-To be expanded
+The week before the release is dedicated to planning the next release.
+Planning accounts for [tasks](#tasks) and [bugs](#bugs) which we intend to solve in the next version.
+Planning meetings will be held, focusing on the [Backlog board].
+
+For a [task](#tasks) to be eligible for the next version, it needs to fulfill these requirements:
+
+* Be part of the [Backlog board].
+* Have a `next-version-candidate` label.
+* Have a description detailing **what** is the issue and optionally **how** is it going to be solved.
+* Have an apropriate sizing label, according to amount of work expected for a single person to completely deliver the task:
+  * *Small*: Work is contained in one sprint and is expected to take less than half the sprint.
+  * *Medium*: Work is contained in one sprint and is expected to take most of the sprint.
+  * *Large*: Work is contained within a release (two-three sprints).
+  * *Extra-Large* = Work can't be contained within a release and would span multiple releases.
+* Any *Large* or *Extra-Large* task must be converted to an [epic](#epics).
+* In case of an [epic](#epics):
+  * Be an issue in the [enhancements] project.
+  * Adhere to the [epic template].
+  * Have a high level break down of the **expected** work, corresponding to the "Definition of Done".
+
+The project team will hold planning meetings, led by the project's "scrum master".
+During these meetings, the project team will:
+
+* Prioritize and assign tasks for the next version.
+  * Only tasks adhering to the described requirements will be considered.
+* Transfer tasks to the [next release board](#current-release-board) according to the capacity of the team to deliver it.
+* Remove the `next-version-candidate` label from transferred tasks.
+
+By the end of the pre-planning week, the project team should have a backlog of tasks and can commence working on the design phase.
```

Comment:
I might have missed it, do we have notes on the process to pick up new priority epics (or unplanned tasks with higher priority) in the middle of a cycle?

Reply:
I am not sure if we have it captured, but I think we need to account for:

1. An urgent bug coming from a user in the middle of a sprint
2. A new feature request or unplanned chunk of work in the middle of a sprint

The first case is tricky. I don't know if we want to document something like this, but we really need to acknowledge that there will be bugs and people need to reserve ~20% time for these.

The second case is something we need to be very clear on IMHO; after planning is done, we should avoid adding any unplanned feature work, because there is a cost for doing so. If you think something is really important, I think it should be discussed during sprint boundary day (when reviewing the board before a new sprint) and agree that this is important enough to focus on (and potentially changing other priorities). https://github.com/submariner-io/submariner-website/pull/717/ mentions "Moving epics back to the Backlog, in case they're de-prioritized" under the Grooming call, but maybe we can reward that because this should not be limited to epics only.

@mkolesnik @Jaanki

Reply:
Yes I will add capacity considerations to the doc, I was just thinking about it yesterday.

This section I feel is the heaviest in detail, perhaps I'll restructure it a bit..

Anyhow regarding unplanned work it has indeed been considered and (will be) documented at https://submariner.io/development/contribution-guide/#unplanned-work so lets discuss it on that PR (which is not written yet ;))